### PR TITLE
feat: add ATSPI dependency for Electron >= 5

### PR DESCRIPTION
**Pull request by fcastilloec**

### CODE REVIEW

File: src/dependencies.js [hunk 1]
```diff
@@ -40,6 +40,13 @@ function getUUIDDepends (version, dependencyMap) {
   return semver.gte(version, '4.0.0-beta.1') ? [dependencyMap.uuid] : []
 }
 
+/**
+ * Determine whether libatspi2 is necessary, given the Electron version.
+ */
+function getATSPIDepends (version, dependencyMap) {
+  return semver.gte(version, '5.0.0-beta.1') ? [dependencyMap.atspi] : []
+}
```
Comment (malept) [inline src/dependencies.js]: Can you put this in alphabetical dependency method order?

File: src/dependencies.js [hunk 2]
```diff
@@ -54,11 +61,13 @@ module.exports = {
       dependencyMap.xdgUtils
     ].concat(getGConfDepends(version, dependencyMap))
       .concat(getUUIDDepends(version, dependencyMap))
+      .concat(getATSPIDepends(version, dependencyMap))
```
Comment (malept) [inline src/dependencies.js]: Ditto (although I realize that's more lines changed).

File: src/dependencies.js [hunk 3]
```diff
@@ -54,11 +61,13 @@ module.exports = {
...
   },
   getGConfDepends: getGConfDepends,
   getGTKDepends: getGTKDepends,
   getTrashDepends: getTrashDepends,
   getUUIDDepends: getUUIDDepends,
+  getATSPIDepends: getATSPIDepends,
```
Comment (malept) [inline src/dependencies.js]: Ditto

File: test/dependencies.js [hunk 4]
```diff
@@ -70,6 +71,10 @@ test('getUUIDDepends: returns uuid as of 4.0', t => {
   t.is(dependencies.getUUIDDepends('4.0.0', dependencyMap)[0], dependencyMap.uuid)
 })
 
+test('getATSPIDepends: returns atspi as of 5.0', t => {
+  t.is(dependencies.getATSPIDepends('5.0.0', dependencyMap)[0], dependencyMap.atspi)
+})
```
Comment (malept) [inline test/dependencies.js]: Ditto

**Comment by fcastilloec** [comment]
I forgot to mention about electron 6. I did an audit for electron 6 and it doesn't have any changes compared with electron 5.

**Pull request merged**